docs/src/components/components/card/card.html

```diff
@@ -1,9 +1,8 @@
 {% for org in organizations %}
-<div class="panel-row-boxed">
+<div class="card">
```

Comment:
Using this pattern to show general Card rather than the Overview feature specifically.